routers/web/repo/branch.go

```diff
@@ -61,6 +62,7 @@ func Branches(ctx *context.Context) {
 		(ctx.IsSigned && repo_model.HasForkedRepo(ctx.Doer.ID, ctx.Repo.Repository.ID))
 	ctx.Data["PageIsViewCode"] = true
 	ctx.Data["PageIsBranches"] = true
+	ctx.Data["EnableFeed"] = setting.EnableFeed
```

Comment:
And here, does  #23904  help about the EnableFeed ?

Reply:
Yes, I've reverted that change.